1-js/03-code-quality/04-ninja-code/article.md

```diff
@@ -66,17 +66,17 @@ The great image has no form.
 
 While choosing a name try to use the most abstract word. Like `obj`, `data`, `value`, `item`, `elem` and so on.
 
-- **The ideal name for a variable is `data`.** Use it everywhere where you can. Indeed, every variable holds *data*, right?
+- **The ideal name for a variable is `data`.** Use it everywhere you can. Indeed, every variable holds *data*, right?
```

Comment:
@iliakan This change is grammatically correct. The extra "where" is redundant. And to answer "does it sound good here": yes, this sounds like natural English. It fits your conversational style well, I think :)